### PR TITLE
structure: Add section on rules for links to external sites

### PR DESCRIPTION
**Pull request by sknorr**
(A little more than) we discussed on [Dec 3](https://confluence.suse.com/x/ggBPIw).

### CODE REVIEW

**Review by lproven — APPROVED** [review]

File: xml/docu_styleguide.structure.xml [hunk 1]
```diff
@@ -790,6 +790,43 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+   <listitem>
+    <para>
+     Prefer larger, well-kept sites over blogs that may vanish overnight.
+     If you need to link to smaller blogs, save an archive version of the
+     site at <link xlink:href="https://web.archive.org/"/>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Where possible, avoid linking to sites that feature controversial or
```
Comment (cwickert) [inline xml/docu_styleguide.structure.xml:818]:
```suggestion
     Avoid linking to sites that feature controversial or
```

File: xml/docu_styleguide.structure.xml [hunk 2]
```diff
@@ -790,6 +790,43 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+   In such cases, choose between linking to these sites or including their
+   content in edited form.
+   Adhere to the following guidelines when selecting sites to link to:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Link to credible sources, such as suse.com, upstream projects, or
+     developer sites.
+     Where possible, avoid linking to direct competitors of &suse;.
```
Comment (cwickert) [inline xml/docu_styleguide.structure.xml:805]:
```suggestion
     Avoid linking to direct competitors of &suse;.
```

Comment (sknorr) [inline xml/docu_styleguide.structure.xml:805]: cwickert, "Where possible, " is unnecessary (dito below).

**Review by cwickert — CHANGES_REQUESTED** [review]
Remove "where possible, ". Rest LGTM

File: xml/docu_styleguide.structure.xml [hunk 3]
```diff
@@ -790,6 +790,43 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+     Prefer larger, well-kept sites over blogs that may vanish overnight.
+     If you need to link to smaller blogs, save an archive version of the
+     site at <link xlink:href="https://web.archive.org/"/>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Where possible, avoid linking to sites that feature controversial or
+     political content.
+     Make sure to also check the site with your ad-blocker disabled, to see
```
Comment (cwickert) [inline xml/docu_styleguide.structure.xml:820]: I suggest to remove that sentence. Site owners often cannot control what ads get displayed.

File: xml/docu_styleguide.structure.xml (hunk 2 above)
Comment (cwickert) [inline xml/docu_styleguide.structure.xml:805]:
```suggestion
     Avoid linking to direct competitors of &suse;.
```

File: xml/docu_styleguide.structure.xml (hunk 1 above)
Comment (cwickert) [inline xml/docu_styleguide.structure.xml:818]:
```suggestion
     Avoid linking to sites that feature controversial or
```

**Review by cwickert — CHANGES_REQUESTED** [review]
Remove "Where possible, " and the sentence about ads. Rest LGTM.

**Review by lproven — APPROVED** [review]

File: xml/docu_styleguide.structure.xml (hunk 3 above)
Comment (cwickert) [reply xml/docu_styleguide.structure.xml:820]: To clarify: The ads usually depend on the client visiting the page, not on the server hosting it.

**Review by taroth21 — CHANGES_REQUESTED** [review]
I'm fine with Christoph's suggestions below and would suggest to merged after the suggestions are in.

File: xml/docu_styleguide.structure.xml [hunk 4]
```diff
@@ -790,6 +790,43 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+     if there are inappropriate ads.
```
Comment (sknorr) [inline xml/docu_styleguide.structure.xml:821]:
```suggestion
```

File: xml/docu_styleguide.structure.xml [hunk 5]
```diff
@@ -790,6 +790,43 @@ will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
...
+    <para>
+     Prefer larger, well-kept sites over blogs that may vanish overnight.
+     If you need to link to smaller blogs, save an archive version of the
+     site at <link xlink:href="https://web.archive.org/"/>.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Where possible, avoid linking to sites that feature controversial or
+     political content.
```
Comment (sknorr) [inline xml/docu_styleguide.structure.xml:819]:
```suggestion
     Avoid linking to sites that feature controversial or political content.
```

File: xml/docu_styleguide.structure.xml (hunk 3 above)
Comment (sknorr) [reply xml/docu_styleguide.structure.xml]: Sentence removed -> resolved.

File: xml/docu_styleguide.structure.xml (hunk 2 above)
Comment (sknorr) [reply xml/docu_styleguide.structure.xml]: Done -> resolved

File: xml/docu_styleguide.structure.xml (hunk 1 above)
Comment (sknorr) [reply xml/docu_styleguide.structure.xml]: Done -> resolved.

**Pull request merged**